src/main/java/org/cryptomator/frontend/fuse/ReadOnlyAdapter.java

```diff
@@ -331,8 +328,8 @@ protected int getErrorCodeForGenericFileSystemException(FileSystemException e, S
 //			LOG.warn("{} {} failed, name too long.", opDesc);
 //			return -ErrorCodes.ENAMETOOLONG();
 //		} else {
-			LOG.error(opDesc + " failed.", e);
-			return -errno.eio();
+		LOG.error(opDesc + " failed.", e);
+		return -errno.eio();
```

Comment:
unrelated

Reply:
done in https://github.com/cryptomator/fuse-nio-adapter/pull/80/commits/a4e330bb0b8154847c9ee048f01b2788ad052a25